dev/PushNotifications/PushNotificationsLongRunningTask/platform.cpp

```diff
@@ -90,11 +97,12 @@ STDMETHODIMP_(HRESULT __stdcall) NotificationsLongRunningPlatformImpl::Unregiste
     auto lock = m_lock.lock_shared();
     THROW_HR_IF(WPN_E_PLATFORM_UNAVAILABLE, m_shutdown);
 
-    const std::wstring appId = GetAppIdentifier(processName);
+    const std::wstring appId{ GetAppIdentifier(processName) };
```

Comment:
If this is empty, shouldn't we bail out of the function?

Reply:
I think we should and in addition probably need to call into this code before bailing out right?

if (m_notificationListenerManager.IsEmpty())
    {
        m_lifetimeManager.Setup();
    }

Reply:
This code i believe basically sets up LRP to shutdown in x seconds if there are no long running sinks registered. Please confirm though. @danielayala94 added this earlier.